apps/addons/buttons.py

```diff
@@ -90,7 +90,11 @@ def __init__(self, addon, app, lang, version=None, show_contrib=True,
                  detailed=False, impala=False):
         self.addon, self.app, self.lang = addon, app, lang
         self.latest = version is None
-        self.version = version or addon.current_version
+        self.latest_beta = version == 'beta'
+        if version == 'beta':
+            self.version = addon.current_beta_version
+        else:
+            self.version = version or addon.current_version
```

Comment:
Mmm, this works, but I'd rather not have a parameter that accepts either a version, `None` or `'beta'`. How about adding a new `beta=False` parameter, and then:
```
self.beta = beta
if self.beta:
    self.version = addon.current_beta_version
else:
    self.version = version or addon.current_version
```

You would then call the `install_button` helpers below with `version=None, beta=True`.

What do you think?